tutorials/save-query-output.md

```diff
@@ -5,4 +5,95 @@ nav_order: 2
 parent: Tutorials
 ---
 
-## Save the Output of a Query
+GlareDB provides a number of options to save the output of your queries,
+including GlareDB tables, temp tables, and views, and external files in a
+variety of formats.
+
+## Creating a GlareDB Table from a File
+
+For instance, say you have found a CSV file of user data on GitHub that you'd like
+to save to our GlareDB instance. You can run:
+
+```sql
+CREATE TABLE my_user_data AS
+SELECT * FROM read_csv(
+    'https://github.com/GlareDB/glaredb/raw/main/testdata/csv/userdata1.csv'
+);
+```
+
+to create a table from the CSV file.
+
+<!--TODO: add link below -->
+
+For more information, see the page on `CREATE TABLE`
+
+## Creating a GlareDB View from a Query
+
+Now, if you'd like to create an unmaterialized view of those users who have
+comments in their record, you can do so with:
+
+```sql
+CREATE VIEW users_with_comments AS
+SELECT * FROM my_user_data WHERE
+comments IS NOT NULL;
+```
+
+<!--TODO: add link below -->
+
+For more information, see the page on `CREATE VIEW`
+
+## Copying the Results of a Query to a File
+
+### Copying to a local Excel File
+
+Perhaps after running a query, you've found something noteworthy in the data
+about users in the United States. Now you'd like to share the results with a
+colleague who is great with Excel, but is not as comfortable with SQL. You can
+run:
+
+```sql
+COPY
+(SELECT * FROM my_user_data WHERE 'country' = 'United States')
+TO './US_users.xlsx
+```
+
+This will save a copy of the data as an Excel file that you can share with your
+colleagues. You can save the data in a variety of other formats, too, including
+Parquet, CSV, and Delta.
+
+### Copying to Cloud Storage
```

Comment:
**future note/informational**: We would benefit from a tutorial on credential objects, to simplify queries like this